python/ql/src/Security/CWE-732/WeakFilePermissions.ql

```diff
@@ -0,0 +1,42 @@
+/**
+ * @name Overly permissive file permissions
+ * @description Allowing files to be readable or writable by users other than the owner may allow sensitive information to be accessed.
+ * @kind problem
+ * @id py/overly-permissive-file
+ * @problem.severity warning
+ * @sub-severity high
+ * @precision medium
+ * @tags external/cwe/cwe-732
+ *       security
+ */
+import python
+
+bindingset[p]
```

Comment:
I feel like it should be possible to do this more elegantly by defining a `Permission` subclass of `int` that only contains the values `0` to `7` (but if we're only using permissions for this query, it hardly matters).